pkg/openwsn/patches/0011-kernel-add-riot-based-scheduler.patch

```diff
@@ -0,0 +1,231 @@
+From 76c9b35489a94d80e43457210479386ea57452e0 Mon Sep 17 00:00:00 2001
```

Comment:
Just a small question:  Is it not possible to collect all RIOT related additions in proper *.[c|h] files and put them, e.g., under a `contrib/` folder in the package?  Managing patch files that are relatively large is IMO counterintuitive ..  Especially if you have to look at patches of patches (as below) (:

Reply:
Hm yes I think for this one I could do it that way, and should be easier, could catch! But all other ones are small patches in code, so it could not be done.

Reply:
Did as suggested @cgundogan!

Reply:
> Did as suggested @cgundogan!

wow that was fast (: and looks much cleaner now.

Too bad the other patch file cannot be removed. But this probably requires some more modularization on the openwsn side.

Reply:
> Too bad the other patch file cannot be removed. But this probably requires some more modularization on the openwsn side.

I can push the change upstream as well actually, and eventually remove the patch.

Reply:
Opened https://github.com/openwsn-berkeley/openwsn-fw/pull/521, I can remove the patch if it is merged.

Reply:
Merged upstream so patch removed.